index.html

```diff
@@ -221,6 +221,42 @@ <h3>Skills</h3>
               
           </div>
         </section>
+
+        <section class="contactForm-section ">
+          
+
+          <h2>Contact me</h2>
+          <p>If you have an application you<br>
+              are interested in developing, a<br>
+              feature that you need built or a<br>
+              project that needs coding. I’d<br>
+              love to help with it</p> 
+          <form  class="contact-form" action="https://formspree.io/f/mgedjayo" method="post">
+            <ul class="contact-input-form">
+              <li>              
+                <input class="input" type="text" name="full_name" placeholder="Full name"
+                  maxlength="30" required />
+              </li>
+              <li>
+                <input class="input" type="email" name="email" placeholder="Email address"
+                required /> 
+              </li>
+              <li>
+                <textarea placeholder="Write your message here"
+                 class="txt" name="contact_text" 
+                 cols="30" rows="5"  maxlength="500" required
+                 ></textarea>
+              </li>
+              <li>
+                <button type="submit" class="contact-button">
+                  Get in Touch
+                </button>
```

Comment:
- [ ] As the previous reviewer muntioned, kindly implement Button interactions such as hover, and active (when pressed), so they are styled as shown in the Figma template.

![btn](https://user-images.githubusercontent.com/25227022/177471020-7862cf32-f749-41dc-aa59-140838165e20.png)